Point cloud acquired frame

dNechita opened the pull request.

VHolonec commented on bindings/python/examples/showPointCloud/showPointCloud.py:
```diff
@@ -52,134 +53,201 @@ def transform_image(np_image):
+        if not status:
+            print("system.getAvailableFrameTypes() failed with status: ", status)
+
+        status = cameras[0].setFrameType("lr-qnative")
```
Now on dev-5.1.0 branch this method has another name like setMode

liealex replied:
Solved

VHolonec commented on bindings/python/examples/showPointCloud/showPointCloud.py:
```diff
@@ -52,134 +53,201 @@ def transform_image(np_image):
+        types = []
+        status = cameras[0].getAvailableFrameTypes(types)
+        if not status:
+            print("system.getAvailableFrameTypes() failed with status: ", status)
```
Now on dev-5.1.0 branch this method has another name like getAvailableModes

liealex replied:
Solved

dNechita commented:
> In January we removed this example (#556). It makes sense to keep only the fix.

Looks like we removed the C++ example and not the Python example. So we will keep this.
